src/app/f2f/controllers/checkDetails.js

```diff
@@ -133,6 +134,11 @@ class CheckDetailsController extends DateController {
     return '/done'
   }
   async saveValues(req, res, callback) {
+    let address = req.sessionModel.get("postOfficeAddress")
+    // Remove the postcode that is appended to the address in result.js
+    var postCodeIndex = address?.lastIndexOf(", ");
```

Comment:
Instead of separating with a` ,`, could this be done via replace to search for the `postOfficePostcode` in `postOfficeAddress` pls

Reply:
As a better option, could add a new var postOfficeAddressWithoutCode [here](https://github.com/alphagov/di-ipv-cri-f2f-front/blob/a12869ed4db796421597f72dae52e41704e274c9/src/app/f2f/controllers/results.js#L66) and use it in checkDetails.js to pass through, that would not need any string replace

Reply:
I thought about that but was worried about the extra lines of code! Have changed now